gulpfile.js

```diff
@@ -7,7 +7,13 @@ const postcss = require('gulp-postcss');
 const autoprefixer = require('autoprefixer');
 const browserSync = require('browser-sync').create();
 const path = require('path');
+const imagemin = require('gulp-imagemin');
+const cleanCSS = require('gulp-clean-css');
+const rename = require("gulp-rename");
+const concat = require('gulp-concat');
+const uglify = require('gulp-uglify');
 
+// Test Sass compliler
 gulp.task("sass", function() {
```

Comment:
is this task  a duplication of a new one?

Reply:
Is it meant to be used for development? Can you reuse the same one? And run browser sync from dev folder, just not minify code for development.